Cargo.toml

```diff
@@ -8,8 +8,12 @@ license = "MIT"
 name = "ordered-multimap"
 readme = "README.md"
 repository = "https://github.com/sgodwincs/ordered-multimap-rs"
-version = "0.3.0"
+version = "0.4.0"
 
 [dependencies]
 dlv-list = "0.2.2"
 hashbrown = "0.9.0"
+serde = { version = "1", optional = true }
+
+[features]
+serde-1 = ["serde"]
```

Comment:
Is it convention to have the `-1` suffix to indicate the version?

Reply:
It was a workaround but it's not required any more with cargo dependency renames, I will change it to `serde`.